test/integration/testConnection.js

```diff
@@ -98,8 +99,10 @@ describe('Connection test', function ()
       done();
     });
   });
+
   it('Multiple Client', function (done)
   {
+    this.timeout(15000);
```

Comment:
can you elaborate why you add this?

Reply:
I saw the multiple client test failing with 6000, so I increased the timeout to 15000 and that worked.  I am not sure, if it is to do with just my testing.